vsphere/internal/helper/contentlibrary/content_library_helper.go

```diff
@@ -191,13 +191,18 @@ func CreateLibraryItem(c *rest.Client, l *library.Library, name string, desc str
 
 	isOva := false
 	isLocal := true
+	isIso := false
 
 	if strings.HasPrefix(file, "http") {
 		isLocal = false
 	}
 	if strings.HasSuffix(file, ".ova") {
 		isOva = true
 	}
+	if strings.HasSuffix(file, ".iso") {
+		isIso = true
+		isOva = false
```

Comment:
I'm not sure I understand why `isOva = false` is used here if it's already defined as false.

Reply:
You're right, it's unnecessary. I'm pushing a new version with this fixed.